lambdas/SaveSnippetToS3/lambda_function.py

```diff
@@ -7,8 +7,15 @@
 import string
 import boto3 # AWS SDK for Python
 
-s3     = boto3.client('s3', 'us-west-2')
+s3 = boto3.client('s3', 'us-west-2')
 client = boto3.client('lambda')
+s3_resource = boto3.resource('s3')
+
+# Returns time the snippet was last modified
+def get_last_modified(bucket, user_id, snippet_key):
```

Comment:
How about you just take the timestamp off the object that is returned [here](https://github.com/maryvilledev/codesplain-lambdas/blob/1db9626d783c11e0d849bc934ee2d4259d42fd22/lambdas/SaveSnippetToS3/lambda_function.py#L65) that is currently being ignored. https://boto3.readthedocs.io/en/latest/reference/services/s3.html#S3.Bucket.put_object